ports/dbg-macro/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "dbg-macro",
-  "version-string": "2019-07-11",
-  "port-version": 1,
+  "version-string": "2021-08-08",
```

```suggestion
  "version-date": "2021-11-02",
```
We prefer to use the date that you accept the commit. https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/manifest-files.md